docker-compose.yml

```diff
@@ -35,19 +101,25 @@ services:
 
   listener:
     build:
-      context: $PWD
+      context: $PWD/docker-files/listener
     image: candig/listener-service:${LISTENER_VERSION:-latest}
     init: true
-    #volumes:
-      #- $PWD:/srv
-    ports:
-      - "${LISTENER_PORT:-8080}:8080"
+    volumes:
+      - $PWD:/srv
+    #ports:
+    #  - "${LISTENER_PORT:-8080}:8080" # This doesn't need to be open because minio uses service name, this also conflicts with traefik
     environment:
+      - MINIO_DATA_DIR=${MINIO_DATA_DIR:-/data}
+      - MINIO_PROCESSED_DIR=${MINIO_PROCESSED_DIR:-samples/processed}
       - MINIO_REGION=${MINIO_REGION:-us-east-1}
       - MINIO_DOMAIN=${MINIO_DOMAIN:-localhost}
+      - MINIO_UI_PORT=${MINIO_UI_PORT:-9000}
       - MINIO_ACCESS_KEY=${MINIO_ACCESS_KEY:-miniotest}
       - MINIO_SECRET_KEY=${MINIO_SECRET_KEY:-miniotest}
       - FLASK_ENV=${LISTENER_ENV:-development}
+      - DOS_PORT=${DOS_PORT:-8080}
```

Comment:
see other comments regarding changes to `DOS_PORT`

Reply:
Solved in e3b0539